src/flb_log_event_encoder.c

```diff
@@ -349,3 +349,34 @@ int flb_log_event_encoder_append_root_values_unsafe(
 
     return result;
 }
+
+const char *flb_log_event_decoder_strerror(int result)
```

Comment:
The same requests from  `flb_log_event_encoder_strerror` apply to this function.

Reply:
I modified it.